src/platform/datapath_winkernel.c

```diff
@@ -1463,7 +1463,11 @@ CxPlatSocketCreateUdp(
             WSK_FLAG_DATAGRAM_SOCKET,
             Binding,
             &Datapath->WskDispatch,
+#ifdef QUIC_OWNING_PROCESS
+            Config->OwningProcess,
```

Comment:
Does the socket abstraction need to store this and `AddRef`/`Release` on it?  I am assuming here that WSK will `AddRef` if it needs to keep the process handle around. 
Since this call completes inline, I don't `AddRef`/`Release`, since the Configuration still owns the pointer.

Reply:
You can remove the `#ifdef` check, and I don't think the socket abstraction needs to store this. Maintaining it on the configuration is enough.

Reply:
I have the parameter on the `UdpConfig` also wrapped in the `#ifdef` so it doesn't confuse usage when that feature is not enabled. So this should stay.

Reply:
I disagree. You only need the ifdef in the cases where that source might not have it. The winkernel.c code has it 100% of the time.